lms/templates/student_account/account_settings.underscore

```diff
@@ -1,5 +1,16 @@
 <main id="main" aria-label="Content" tabindex="-1">
 <div class="account-settings-container">
+    <% if (message) { %>
+        <div class="account-settings-message">
+            <div id="beta-language-message" class="alert-message warning" aria-live="assertive" aria-hidden="true">
```

Comment:
Please add role="alert" here (line 5).  Also, aria-hidden should be true only when there is no message, or false (or absent) when there is a message. So I don't know if aria-hidden needs to be here at all.

Reply:
@wittjeff Added role and removed aria hidden, kindly take another look.